src/lib/atoms/BusyLoader/BusyLoader.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { shallow, mount, render, ShallowWrapper, ReactWrapper, MountRendererProps } from 'enzyme';
+import BusyLoader, { IBusyLoaderProps } from './index';
+
+import { Icon } from '../../../index';
+import BubbleLoader from './BubbleLoader';
+
+const shallowComponent = (props?: IBusyLoaderProps): ShallowWrapper => shallow(<BusyLoader {...props} />);
```

Comment:
Please make one function out of these functions and initialize it in beforeEach.